test/utils/utils.ts

```diff
@@ -0,0 +1,69 @@
+import axios from "axios";
+import assert, { AssertionError } from "assert";
+import { existsSync, rmSync } from "fs";
+import shell from "shelljs";
+import { StarknetPluginError } from "../../src/starknet-plugin-error";
+import { DEVNET_URL } from "../constants/constants";
+
+export function exec(cmd: string) {
+    const result = shell.exec(cmd);
+    if (result.code !== 0) {
+        throw new StarknetPluginError(`Command ${cmd} failed.\n${result.stderr}`);
+    }
+
+    return result;
+}
+
+export function contains(output: string, pattern: string) {
```

Comment:
Rename this to `assertContains` and throw an assertion error instead of starknet plugin error.

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/251/commits/f869821c5c5153f5e4b8878ca852a3fce03469ad